cpp/arcticdb/storage/mongo/mongo_storage.cpp

```diff
@@ -145,13 +146,19 @@ MongoStorage::MongoStorage(
     OpenMode mode,
     const Config &config) :
     Storage(lib, mode),
-    instance_(MongoInstance::instance()),
-    client_(std::make_unique<MongoClient>(
-        config,
-        ConfigsMap::instance()->get_int("MongoClient.MinPoolSize", 100),
-        ConfigsMap::instance()->get_int("MongoClient.MaxPoolSize", 1000),
-        ConfigsMap::instance()->get_int("MongoClient.SelectionTimeoutMs", 120000))
-        ) {
+    instance_(MongoInstance::instance()) {
+    if(config.use_mock_storage_for_testing()) {
+        ARCTICDB_RUNTIME_DEBUG(log::storage(), "Using Mock Mongo storage");
+        client_ = std::make_unique<MockMongoClient>();
+    }
+    else {
```

Comment:
(nit) we usually have the else on the line above

Reply:
Fixed in [3283f32](https://github.com/man-group/ArcticDB/pull/1395/commits/3283f32c0a323b10940abd25d0b876ca8b57984e)